.github/workflows/ingest-maintained-apps.yml

```diff
@@ -50,7 +50,32 @@ jobs:
           go mod download
           go run cmd/maintained-apps/main.go
 
+      - name: Search for Existing PRs
+        id: search_pr
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const { data: pullRequests } = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open',
+              per_page: 100
```

Comment:
If we get 100 prs opened between runs then we'll fail to close some older ones.  This seems unlikely enough (we don't have 100 prs _total_ right now) that it's not worth implementing scan-and-scroll for this.

Reply:
Yeah and once this is merged we should be closing max 1 PR at a time (since this runs twice a day)